src/main/java/flaxbeard/steamcraft/api/block/BlockSteamTransporter.java

```diff
@@ -14,7 +14,8 @@ public void onBlockPreDestroy(World world, int x, int y, int z, int meta) {
         ISteamTransporter te = (ISteamTransporter) world.getTileEntity(x, y, z);
         if (te != null && te.getNetwork() != null) {
             te.getNetwork().split(te, true);
+            te.refresh();
         }
-        te.refresh();
```

Comment:
This specifically needs to be tested, as there was a really major bug that this line fixed (#294)